Mathlib/MeasureTheory/Order/Group/Lattice.lean

```diff
@@ -22,29 +22,29 @@ variable {α β : Type*} [Lattice α] [Group α] [MeasurableSpace α]
 variable [MeasurableSup α]
 
 @[to_additive (attr := measurability)]
-theorem measurable_oneLePart : Measurable fun x : α ↦ oneLePart x :=
+theorem measurable_oneLePart : Measurable (oneLePart : α → α) :=
   measurable_sup_const _
 
 @[to_additive (attr := measurability)]
-protected theorem Measurable.oneLePart : Measurable fun x ↦ oneLePart (f x) :=
+protected theorem Measurable.oneLePart : Measurable (oneLePart ∘ f) :=
   measurable_oneLePart.comp hf
 
 variable [MeasurableInv α]
 
 @[to_additive (attr := measurability)]
-theorem measurable_leOnePart : Measurable fun x : α ↦ leOnePart x :=
+theorem measurable_leOnePart : Measurable (leOnePart : α → α) :=
   (measurable_sup_const _).comp measurable_inv
 
 @[to_additive (attr := measurability)]
-protected theorem Measurable.leOnePart : Measurable fun x ↦ leOnePart (f x) :=
+protected theorem Measurable.leOnePart : Measurable (leOnePart ∘ f) :=
```

Comment:
I think the eta-expanded version is better for those composition lemmas.

Reply:
You have very specific taste ;)
Ok, I've changed these back.

Reply:
@fpvandoorn wrote a note (about continuity, but the same applies to measurability) in which he explained why `Measurable fun x ↦ leOnePart (f x)` is better than `Measurable (leOnePart ∘ f)` for that type of lemma. https://leanprover-community.github.io/mathlib_docs/notes.html#continuity%20lemma%20statement
It is likely specific to Lean 3 and now obsolete since the continuity and measurability tactics use aesop, but I'm not sure. 

In applications of that lemma, I expect the goal to have the form `Measurable fun x ↦ leOnePart (f x)`  more often that the other. Also the thread https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/Aesop.20and.20Lambdas make me think that we might just need both versions.

@YaelDillies can you tell why you think `Measurable (leOnePart ∘ f)` is better?

Reply:
> @fpvandoorn wrote a note (about continuity, but the same applies to measurability) in which he explained why `Measurable fun x ↦ leOnePart (f x)` is better than `Measurable (leOnePart ∘ f)` for that type of lemma. https://leanprover-community.github.io/mathlib_docs/notes.html#continuity%20lemma%20statement It is likely specific to Lean 3 and now obsolete since the continuity and measurability tactics use aesop, but I'm not sure.

I see. Thanks for the link. 

> In applications of that lemma, I expect the goal to have the form `Measurable fun x ↦ leOnePart (f x)` more often that the other. Also the thread https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/Aesop.20and.20Lambdas make me think that we might just need both versions.

I can add the other spelling as primed version is you think it's necessary.
 
> @YaelDillies can you tell why you think `Measurable (leOnePart ∘ f)` is better?

No, I think Yaël agrees with you. I was the one that was asking for some justification.

Reply:
Oh right, I got confused and switched mentally red and green lines in the diff.
Let's merge that PR as it is now. We'll see if we need other versions of the lemmas as we use them. Given recent discussions on zulip about `fprop`, it's also possible that the tactic we use for measurability changes completely in the near future.